invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
     """Display user dashboard page."""
     if not current_app.config["COMMUNITIES_ENABLED"] or not dashboard_name:
         dashboard_name = current_app.config["_DASHBOARD_ROUTES"][0]
+    prefix = current_user_resources.users_resource.config.url_prefix
```

Comment:
can you please move this view from `deposits.py` to `records.py`? It does not belong here.